### ISSUE TITLE
NVDA announces whitespace characters in canvas fallback content

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#10661]
This might be a chrome issue, as I can not reproduce it on Firefox.

### Steps to reproduce:

```
<h1>browse the following canvas with NVDA</h1>
	<canvas width=320 height=240 style="background:red">
		<h2>Hullo!</h2>
		<p>foo bar!</p>
	</canvas>
```

### Actual behavior:
All tabs and blank lines between elements inside the canvas element are announced as "	" (that's a tab character) and "blank".

### Expected behavior:
Fallback markup inside a canvas element should be handled by NVDA exactly as if they were outside the canvas: White space should not be announced by default.

#### NVDA version:
2019.2.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Chrome Version 79.0.3945.88 (Official Build) (64-bit)
Issues also observed on previous version

**Comment by DrSooom** [comment]
@brennanyoung: Please completely fill out the bug report draft as OS, NVDA and Firefox/Chrome version are currently missing. They are important for reproducing this issue. Thanks.

**Comment by Adriani90** [comment]
This works as expected in Chrome 84 and NVDA Version: alpha-20548,e1f3ba99. Closing as works for me.
Please comment on this if you are still seeing this issue and we can reopen. Thanks.

**Comment by brennanyoung** [comment]
@Adriani90 I am still experiencing this issue on (to date) latest Chrome Version 84.0.4147.89, with latest NVDA (2020.1) on Windows 10. 

I'm also seeing the same issue with latest Microsoft Edge, but **not** on Firefox.

Here's a slightly expanded example, to compare with similar content outside the canvas.
```
<!DOCTYPE html>
<html>
<head>
<title>white space in canvas test</title>
</head>
<body>

<h1>browse the following canvas with a screenreader</h1>
	<canvas width=320 height=240 style="background:red">
		<h2>Hullo</h2>
		<p>foo</p>
	</canvas>
	<div>
		<h3>Goodbye</h3>
		<p>bar</p>
	</div>
</body>
</html>
```

Load the page in Chrome, the **press up/down arrow to browse the canvas content with NVDA**. It announces "blank" (for a sequence of one or more linebreaks) or making null announcements (for a sequence containing tabs). I am not browsing by character, so this should not be happening. Note I am using real tab characters for indentation here, which might not survive in this github snippet. But even without real tabs, the linebreaks are still announced as "blank".

Note that this does **not** happen for the similar markup inside the div.
